iaso/api/export_requests.py

```diff
@@ -38,7 +40,7 @@ def validate(self, data: typing.MutableMapping):
 
     def create(self, validated_data: typing.MutableMapping):
         # FIXME: Temporary fix till we reimplement this in the new task system, timeout old queued requests.
-        REQUEST_TIMEOUT_S = 15 * 60  # 15 minutes
+        REQUEST_TIMEOUT_S = 1  # 15 minutes
```

Comment:
```suggestion
        REQUEST_TIMEOUT_S = 1
```

The comment may not be accurate anymore.